File: integrasjon/src/main/java/no/nav/melosys/integrasjon/dokgen/dto/innvilgelseftrl/VurderingTrygdeavgift.java
package no.nav.melosys.integrasjon.dokgen.dto.innvilgelseftrl;

import com.fasterxml.jackson.annotation.JsonInclude;

@JsonInclude(JsonInclude.Include.NON_EMPTY)
public record VurderingTrygdeavgift(
    TrygdeavgiftInfo norsk,
    TrygdeavgiftInfo utenlandsk,
    boolean selvbetalende,
    String representantNavn
) {
}

File: integrasjon/src/main/kotlin/no/nav/melosys/integrasjon/dokgen/dto/innvilgelseftrl/VurderingTrygdeavgift.kt
package no.nav.melosys.integrasjon.dokgen.dto.innvilgelseftrl

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_EMPTY)
data class VurderingTrygdeavgift(
    val norsk: TrygdeavgiftInfo?,
    val utenlandsk: TrygdeavgiftInfo?,
    val selvbetalende: Boolean,
    val representantNavn: String?
)
